test/Hostnet/HnDependencyInjectionPlugin/Symfony1FallbackTest.php

```diff
@@ -64,10 +86,10 @@ public function testOnKernelExceptionSf2Init()
             ->expects($this->never())
             ->method('fallbackToSymfony1');
 
-        $event1 = $this->buildControllerEvent(array(clone $this->fallback, 'fallbackToSymfony1'));
+        $event1 = $this->buildControllerEvent([clone $this->fallback, 'fallbackToSymfony1']);
```

Comment:
For these square brackets, PHP5.4 is required.

You could remove 5.3 as a target in `.travis.yml`, and require PHP >=5.4 in `composer.json`, or revert back to `array()` notation.